about-us.html

```diff
@@ -0,0 +1,132 @@
+			</button>
+    </div>
+  </nav>   
+  <main id="about-us">
+    <header class="full-width about-header">
+      <div class="wrapper">
+        <h2 class="lato-font secondary-color">Khaled portfolio showcase</h2>
+        <h1 class="lato-font-extraBold secondary-color uppercase">This is my project showcase</h1>
+        <div class="header-content">
+          <p class="lato-font primary-color">Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s</p>
+        </div>
+        <div class="header-info">
+          <h5 class="lato-font-bold primary-color">If you need to reach me <br> do so via my github!</h5>
+					<h5 class="lato-font-bold fourth-color"></h5>
+        </div>
+      </div>
+    </header>
+    <section id="logo" class="white-background full-width">
+      <div class="wrapper">
+        <h2 class="lato-font fourth-color">Learning material <br></h2>
+        <div class="title-deco secondary-background"></div>
+				<h5 class="lato-font primary-color">This is a showcase website for my work and as a means to contact me!</h5>
+       	<div class="big-logo-container">
+					<img src="./assets/line-letter-cv.jpg" alt="CV">
+			 	</div>
+      </div>
+    </section>
+
+    <section id="partners" class="fourth-background full-width">
+      <div class="wrapper">
+        <h2 class="lato-font white-color">Partners</h2>
+        <div class="title-deco secondary-background"></div>
+        <ul id="partners-list">
+					<li class="partner-item">
+						<img src="./assets/company1.jpg" alt="Company1">
+					</li>
+          <li class="partner-item">
+						<img src="./assets/company2.jpg" alt="Company2">
+					</li>
+          <li class="partner-item">
+						<img src="./assets/company3.png" alt="Company3">
+					</li>
+        </ul>
+      </div>
+    </section>
+    <footer class="fourth-background full-width">
+      <div class="wrapper desktop">
+        <div class="logo-container">
+            <img src="./assets/line-letter-cv.jpg" alt="CV">
+            <h4 class="lato-font third-color">My CV</h4>
+        </div>
+        <div class="rigth-container">
+          <h5 class="lato-font third-color">
+            Reach me at glitcher255 on github
+          </h5>
+        </div>
+      </div>
+      <div class="wrapper mobile">
+        <div class="logo-container">
+            <img src="./assets/line-letter-cv.jpg" alt="CV icon">
+            <h4 class="lato-font primary-color">s</h4>
+        </div>
+        <div class="rigth-container">
+        </div>
+      </div>
+    </footer>
+  </main>
```

Comment:
- [ ] **[Optional] Your have done a great job so far 🏅 🏅** 
   However, the is a missing part on your About page as we were requested to replicate the Original design, it would be better too to add all the sections to your project 👍 👍 

**Missing Part**
![image](https://user-images.githubusercontent.com/102757126/210867204-3e41e063-1e0a-4f33-8803-49986fed97ff.png)

Reply:
Many thanks